src/module-loader/dotnet/src/MorganStanley.ComposeUI.ModuleLoader.Abstractions/IModuleCatalogue.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Morgan Stanley makes this available to you under the Apache License,
+// Version 2.0 (the "License"). You may obtain a copy of the License at
+// 
+//      http://www.apache.org/licenses/LICENSE-2.0.
+// 
+// See the NOTICE file distributed with this work for additional information
+// regarding copyright ownership. Unless required by applicable law or agreed
+// to in writing, software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+// or implied. See the License for the specific language governing permissions
+// and limitations under the License.
+
+namespace MorganStanley.ComposeUI.ModuleLoader;
+
+public interface IModuleCatalogue
```

Comment:
Is it Catalog or Catalogue?

Reply:
The original file was called IModuleCatalogue: https://github.com/morganstanley/ComposeUI/blob/main/prototypes/multi-module-prototype/src/module-loader/dotnet/ModuleLoader/Modules/IModuleCatalogue.cs

Reply:
Reading the code I agree that we mostly use US English (e.g.: canceled), so I'm happy to rename this to IModuleCatalog